timelock-agent/src/main/java/com/palantir/timelock/paxos/DbBoundTimestampCreator.java

```diff
@@ -49,15 +52,18 @@ public DbBoundTimestampCreator(KeyValueServiceConfig kvsConfig) {
                 Optional::empty,
                 Optional.of(leaderConfig),
                 Optional.empty(),
-                Optional.of(AtlasDbConstants.LEGACY_TIMELOCK_TIMESTAMP_TABLE),
+                Optional.of(getTimestampCreationParameters(client)),
                 AtlasDbConstants.DEFAULT_INITIALIZE_ASYNC,
                 AtlasDbFactory.THROWING_FRESH_TIMESTAMP_SOURCE);
 
         TimestampService timestampService = atlasFactory.getManagedTimestampService();
-        Preconditions.checkArgument(timestampService instanceof TimestampManagementService,
-                "The timestamp service is not a managed timestamp service.");
 
         return () -> new DelegatingManagedTimestampService(timestampService,
                 (TimestampManagementService) timestampService);
     }
+
+    @VisibleForTesting
+    static DbTimestampCreationSetting getTimestampCreationParameters(Client client) {
+        return DbTimestampCreationSettings.multipleSeries(Optional.empty(), TimestampSeries.of(client.value()));
```

Comment:
Is it safe to just switch here?

Reply:
Yes. This isn't currently used in production anywhere.